WalletWasabi.Fluent/Controls/LineChart.cs

```diff
@@ -298,13 +298,11 @@ private void SetStateAreaPoints(LineChartState state)
 			{
 				state.Points = new Point[xAxisValues.Count];
 
-				using (var enumerator = xAxisValuesEnumerable.GetEnumerator())
+				using var enumerator = xAxisValuesEnumerable.GetEnumerator();
+				for (var i = 0; i < yAxisValuesScaled.Count; i++)
 				{
-					for (var i = 0; i < yAxisValuesScaled.Count; i++)
-					{
-						enumerator.MoveNext();
-						state.Points[i] = new Point(enumerator.Current, yAxisValuesScaled[i]);
-					}
+					enumerator.MoveNext();
```

Comment:
Are we sure there is always anext?

Reply:
I don't know, I didn't touch this part.

Reply:
Review with whitespace changes off https://github.com/zkSNACKs/WalletWasabi/pull/5677/files?diff=unified&w=1